doc/classes/KinematicBody2D.xml

```diff
@@ -24,7 +24,7 @@
 			<return type="Vector2">
 			</return>
 			<description>
-				Returns the linear velocity of the floor at the last collision point. Only valid after calling [method move_and_slide] or [method move_and_slide_with_snap] and when [method is_on_floor] returns [code]true[/code].
+				Returns the linear velocity of the floor at the last collision point. Only valid after it calls [method move_and_slide] or [method move_and_slide_with_snap] and when [method is_on_floor] returns [code]true[/code].
 			</description>
```

Comment:
To me calling makes more sense. With your change it's not clear what the "it" refers to.
This value is updated by the functions `method move_and_slide` and `move_and_slide_with_snap`; so the value is only valid after calling them.